packages/@aws-cdk/aws-redshift/lib/cluster.ts

```diff
@@ -474,6 +474,10 @@ export class Cluster extends ClusterBase {
     this.singleUserRotationApplication = secretsmanager.SecretRotationApplication.REDSHIFT_ROTATION_SINGLE_USER;
     this.multiUserRotationApplication = secretsmanager.SecretRotationApplication.REDSHIFT_ROTATION_MULTI_USER;
 
+    if (props.loggingBucket && !props.loggingKeyPrefix) {
```

Comment:
If these props need to be both or neither, I'd prefer we take `loggingBucket` and `loggingKeyPrefix` and make them into an interface that is optional, with both fields being required. I recognize that this is a breaking change, but I think it's the better contract overall. Since this is an experimental module, we can do this without a feature flag.